store/src/main/java/com/nytimes/android/external/store/base/Store.java

```diff
@@ -11,40 +11,43 @@
  * a {@link com.nytimes.android.external.store.base.impl.StoreBuilder StoreBuilder}
  * will return an instance of a store
  * <p>
- * A {@link com.nytimes.android.external.store.base.Store  Store} can
- * {@link com.nytimes.android.external.store.base.Store#get(BarCode) Store.get() } cached data or
- * force a call to {@link com.nytimes.android.external.store.base.Store#fetch(BarCode) Store.fetch() }
+ * A {@link Store  Store} can
+ * {@link Store#get(BarCode) Store.get() } cached data or
+ * force a call to {@link Store#fetch(BarCode) Store.fetch() }
  * (skipping cache)
  */
-public interface Store<T> {
+public interface Store<T> extends com.nytimes.android.external.store.base.experimental.Store<T, BarCode> {
```

Comment:
renaming would add to breaking changes, I'll just throw a deprecated tag up here